python/graphscope/interactive/query.py

```diff
@@ -32,11 +34,15 @@ class InteractiveQuery(object):
     """`InteractiveQuery` class, is a simple wrapper around
     `Gremlin-Python <https://pypi.org/project/gremlinpython/>`_,
     which implements Gremlin within the Python language.
-    It also can expose gremlin endpoint which can be used by any other standard gremlin console.
+    It also can expose gremlin endpoint which can be used by
+    any other standard gremlin console, with the method `graph_url()`.
 
     It also has a method called `subgraph` which can extract some fragments
     from origin graph, produce a new, smaller but concise graph stored in vineyard,
     which lifetime is independent from the origin graph.
+
+    Use `execute()` to submit a script,
+    ans use `traversal_source()` to get a `GraphTraversalSource` for further traversal.
```

Comment:
`ans` -> `and`.

Could you please use a more readable line-breaks in comment? The following would be better:

```
Use `execute()` to submit a script, and use `traversal_source()` to get a `GraphTraversalSource`
for further traversal.
```

That doesn't affect the generated docs but will benifit the readability of source code. It would be great to think you are writing a document or paper when writing docstring in comments.

Reply:
Yes, I already noticed the typo and have a fix locally, and I'm thinking combine the typo fixes with other comments to avoid another unnecessary CI.